CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+  * Examples: `LibMedia`, `WebContent`, `CI`, `AK`, `RequestServer`, `js`
+  * Don't use a category like "`Libraries`" or "`Utilities`", except for generic changes that affect a large portion of code within these directories.
+  * Don't use specific component names, e.g. C++ class names, as the category either - mention them in the summary instead. E.g. `LibGUI: Brief description of what's being changed in FooWidget` rather than `FooWidget: Brief description of what's being changed`
+  * Several categories may be combined with `+`, e.g. `LibJS+LibWeb+Browser: ...`
+* Write the commit message subject line in the imperative mood ("Foo: Change the way dates work", not "Foo: Changed the way dates work").
+* Write your commit messages in proper English, with care and punctuation.
+* Amend your existing commits when adding changes after a review, where relevant.
+* Mark each review comment as "resolved" after pushing a fix with the requested changes.
+* Add your personal copyright line to files when making substantive changes. (Optional but encouraged!)
+* Check the spelling of your code, comments and commit messages.
+
+**Don't:**
+
+* Submit changes that are incompatible with the project licence (2-clause BSD.)
+* Touch anything outside the stated scope of the PR.
+* Iterate excessively on your design across multiple commits.
+* Use weasel-words like "refactor" or "fix" to avoid explaining what's being changed.
+* End commit message subject lines with a period.
+* Include commented-out code.
+* Attempt large architectural changes until you are familiar with the project and have worked on it for a while.
+* Engage in excessive "feng shui programming" by moving code around without quantifiable benefit.
+* Add jokes or other "funny" things to user-facing parts of the project.
+
+## On usage of AI and LLMs
+
+Usage of AI assistance is usually fine, but you are responsible for making sure the quality of the output is up to the
+standards of the project. Currently, AI-generated output is often too verbose and its quality is subpar compared to what
+most human contributors produce.
+
+Do not use an AI or LLM to generate changes, respond to issues or interact with PRs without holding the output to the
+same standards as human-written content.
+
+## Pull Request Q&A
+
+### I've submitted a PR and it passes CI. When can I expect to get some review feedback?
+
+While unadvertised PRs may get randomly merged by curious maintainers, you will have a much smoother time if you engage with the community on Discord.
+
+### If my PR isn't getting attention, how long should I wait before pinging one of the project maintainers?
+
+Ping them right away if it's something urgent! If it's less urgent, advertise your PR on Discord (`#code-review`) and ask if someone could review it.
+
+### Is Discord the place to ask for review help, or is GitHub preferred?
```

Comment:
the note on stalebot was deleted here, but can be added back if we turn it on  for this repo.